.cirrus.yml

```diff
@@ -41,21 +41,74 @@ docker_builder:
 
 task:
   name: Release (Dry Run)
-  only_if: $CIRRUS_PR != ''
-  depends_on:
-    - Lint
-    - Tests
+  only_if: $CIRRUS_TAG == ''
   container:
     image: goreleaser/goreleaser:latest
   release_script: goreleaser build --snapshot
   binaries_artifacts:
     path: "dist/cirrus_*/cirrus*"
 
+task:
+  name: Release Apple Silicon (Dry Run)
+  only_if: $CIRRUS_TAG == ''
+  env:
+    GITHUB_TOKEN: ENCRYPTED[!98ace8259c6024da912c14d5a3c5c6aac186890a8d4819fad78f3e0c41a4e0cd3a2537dd6e91493952fb056fa434be7c!]
```

Comment:
This doesn't seem to be required for the dry-run task.

Reply:
Fixed in b8db09f6caab7badf9989d71c4caff2cfe66f750